index.js

```diff
@@ -23,15 +24,15 @@ logger.info(
   process.version
 )
 
-if (require.cache.__NR_cache) {
+if (require.cache[symbols.cache]) {
   logger.warn(
     'Attempting to load a second copy of newrelic from %s, using cache instead',
     __dirname
   )
-  if (require.cache.__NR_cache.agent) {
-    require.cache.__NR_cache.agent.recordSupportability('Agent/DoubleLoad')
+  if (require.cache[symbols.cache].agent) {
```

Comment:
We document this usage in `function-wrappming.md`: `const agent = require.cache[symbols.cache]`, the agent is one part that may or may not be on the `API` object stored on `require.cache[symbols.cache]`.

I think it's fine to leave this as-is?

Reply:
OIC, n/m

Reply:
Hm, but on second thought those docs really are wrong, that's grabbing the API, not the agent, and this is all internal anyway, so yeah, I guess we only need the agent. I'm afraid of breaking API too much here, though.

Reply:
Okay, sorry for the back-and-forth with myself here, but I think it does make sense to leave everything as-is here (but fix the docs to `const agent = require.cache[symbols.cache].agent`). We are using the main cache just to see if we've loaded anything at all, possibly only the stub API in case the agent couldn't be loaded. That cached API may or may not have an agent, and in case it does, we can record a metric about double-loading.